docs/requirement-sets/word/word-api-desktop-requirement-set.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Word JavaScript API desktop-only requirement set
+description: Details about the WordApiHiddenDocument requirement set.
+ms.date: 06/24/2022
+ms.prod: word
+ms.localizationpriority: medium
+---
+
+# Word JavaScript API desktop-only requirement set
+
+The `WordApiHiddenDocument` requirement set is a special requirement set that includes features that are only available for Word on Windows and on Mac. APIs in this requirement set are considered to be production APIs for the Word application on Windows and on Mac. They follow [Microsoft 365 developer support policies](/office/dev/add-ins/publish/maintain-breaking-changes). `WordApiHiddenDocument` APIs are considered to be "preview" APIs for other platforms (web, iPad) and may not be supported by any of those platforms.
+
+When APIs in the `WordApiHiddenDocument` requirement set are supported across all platforms, they will added to the next released requirement set (`WordApi 1.[NEXT]`). Once that new requirement is public, those APIs will be removed from `WordApiHiddenDocument`. Think of this as a similar promotion process to an API moving from preview to release.
```

Comment:
I don't know if this is true for desktop-only API sets, for the reasons I mentioned above. @jeremy-msft?

Reply:
Works for me!